thorlcr/activities/lookupjoin/thlookupjoinslave.cpp

```diff
@@ -1565,6 +1566,7 @@ class CInMemJoinBase : public CSlaveActivity, public CAllOrLookupHelper<HELPER>,
             CAsyncCallStart asyncLeftStart(std::bind(&CInMemJoinBase::startLeftInput, this));
             try
             {
+                ActivityTimer s(slaveTimerStats, timeActivities);
```

Comment:
Unless I've missed something I think this should move to head of this start() function.